### ISSUE TITLE
Incompatible shapes: [654] vs. [436]

### ISSUE THREAD

**Issue by ramesh720** [xuanjihe/speech-emotion-recognition#25]
i am running model.py
at this step
loss, train_acc = sess.run([cost,accuracy],feed_dict = {X:valid_data, Y:valid_label,is_training:False, keep_prob:1})
i am getting error
my valid _label shape is (436,4)
my valid _data shape is (436,4)

to calculate loss we need Ylogits . but logits shape is (654,4)
logits and labels must be same size: logits_size=[654,4] labels_size=[436,4]    

can any  one solve  this problem.... Thanking you 

![image](https://user-images.githubusercontent.com/43946769/52198784-00786980-288a-11e9-84a1-053bb1150237.png)

**Comment by xuanjihe** [comment]
print the shape of every tensorflow and you will find the issue!

**Comment by ramesh720** [comment]
('valid_label==', (436, 4))
('valid_data==', (436, 300, 40, 3))


i checked and i printed all shapes . but getting same error. many of them getting the same error as i get. can you please check sir. Thanking you sir

**Comment by xuanjihe** [comment]
now i have updated the code, you can try again!

**Comment by ramesh720** [comment]
thanking you sir for quick reply.
sir you need to change  2 things
 load_validdata(FLAGS.traindata_path) from load_traindata(FLAGS.traindata_path) in model,py line 61
and
    gru, alphas = attention(outputs1,1, return_alphas=True) from     gru, alphas = attention(outputs1, return_alphas=True) in arcnn1.py line 150
